src/components/tile.jsx

```diff
@@ -0,0 +1,40 @@
+import brand from "../assets/brand.png"
```

Comment:
Ook hier kun je het if-else statement voor de afbeeldingen weer weglaten en deze gewoon in App.jsx aan de image property meegeven. 

In App.jsx krijg je dan:
```suggestion
import brand from "../assets/brand.png"

<Tile img={brand} />

```